### PR TITLE
add rennen and momentmal option in schwimmen/hunger file (german)

### PR DESCRIPTION
**Pull request by JuleEF**

### CODE REVIEW

**Comment by JuleEF** [comment]
Please review this, @casey-collab

File: german/schwimmen/hunger/hunger.md [hunk 1]
```diff
@@ -2,3 +2,5 @@ Alle schauen sich um. Wo bekommt man jetzt möglichst schnell was zu essen her?
 Im Schwimmbad hat man nur eine Möglichkeit. Am Kiosk hinten rechts.
 Also hin dort!
 
+[Moment mal - Kiosk?](momentmal/momentmal.md)
+[Renn so schnell wie möglich, damit du der erste bist.](rennen/rennen.md)
```
Comment (casey-collab) [inline german/schwimmen/hunger/hunger.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by JuleEF** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**